internal/gatewayapi/translator.go

```diff
@@ -1519,7 +1520,7 @@ func isValidHostname(hostname string) error {
 }
 
 func irStringKey(gateway *v1beta1.Gateway) string {
-	return fmt.Sprintf("%s-%s", gateway.Namespace, gateway.Name)
+	return utils.GetHashedName(fmt.Sprintf("%s-%s", gateway.Namespace, gateway.Name))
```

Comment:
any specific reason to add this ?

Reply:
@arkodgis ^ b/c we should only be hashing the ns/name at the Infra Manager?

Reply:
yeah, my comment is not a blocker, but I see the need to hash as something thats being done to be conformant within k8s (specific platform), not affecting any other components

Reply:
Removed this bit and now only the service/deployment/configmap/serviceaccount are hashed.